cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -42,7 +42,7 @@ angular.module(PKG.name + '.commons')
           'ng-trim': 'false',
           'ng-model': 'model',
           'rows': '{{myconfig["widget-attributes"].rows}}',
-          placeholder: '{{::myconfig["widget-attributes"].default}}'
+          placeholder: '{{::myconfig["widget-attributes"].placeholder}}'
```

Comment:
We should have this backward compatible. If there are user plugins that has widget jsons using textarea or text widgets with default widget attributes, then those might suddenly not show any placeholders.

```
...
placeholder: '{{::myconfig["widget-attributes].placeholder || ::myconfig["widget-attributes"].default }}'
...
```